docs/en/observability/synthetics-reference/lightweight-config/http.asciidoc

```diff
@@ -0,0 +1,305 @@
+:hardbreaks-option:
+
+[%header]
+|===
+| Option (type) | Description
+
+// hosts
+| [[monitor-http-hosts]] *`hosts`*
+(list of <<synthetics-lightweight-data-string,string>>s)
+a| *Required*. A list of URLs to ping.
+
+////////////////////////
+max_redirects
+////////////////////////
+| [[monitor-http-max_redirects]] *`max_redirects`*
+(<<synthetics-lightweight-data-numbers,number>>)
+a| The total number of redirections Heartbeat will follow. By default, Heartbeat will not follow redirects, but will report the status of the redirect. If set to a number greater than `0`, Heartbeat will follow that number of redirects.
+
+When this option is set to a value greater than `0`, the `monitor.ip` field will no longer be reported, as multiple DNS requests across multiple IPs may return multiple IPs. Fine grained network timing data will also not be recorded, as with redirects that data will span multiple requests. Specifically the fields `http.rtt.content.us`, `http.rtt.response_header.us`, `http.rtt.total.us`, `http.rtt.validate.us`, `http.rtt.write_request.us` and `dns.rtt.us` will be omitted.
+
+*Default*: `0`
+
+////////////////////////
+proxy_url
+////////////////////////
+| [[monitor-http-proxy_url]] *`proxy_url`*
+(<<synthetics-lightweight-data-string,string>>)
+a| The HTTP proxy URL. This setting is optional.
+
+*Example*:
+
+[source,yaml]
+http://proxy.mydomain.com:3128
+
+////////////////////////
+username
+////////////////////////
+| [[monitor-http-username]] *`username`*
+(<<synthetics-lightweight-data-string,string>>)
+a| The username for authenticating with the server. The credentials are passed with the request. This setting is optional.
+
+You need to specify credentials when your `check.response` settings require it. For example, you can check for a 403 response (`check.response.status: [403]`) without setting credentials.
+
+////////////////////////
+password
+////////////////////////
+| [[monitor-http-password]] *`password`*
+(<<synthetics-lightweight-data-string,string>>)
+a| The password for authenticating with the server. This setting is optional.
+
+////////////////////////
+ssl
+////////////////////////
+| [[monitor-http-ssl]] *`ssl`*
+({heartbeat-ref}/configuration-ssl.html[SSL])
+a| The TLS/SSL connection settings for use with the HTTPS endpoint. If you don't specify settings, the system defaults are used.
+
+*Example*:
+
+[source,yaml]
+----
+- type: http
+  id: my-http-service
+  name: My HTTP Service
+  hosts: ["https://myhost:443"]
+  schedule: '@every 5s'
+  ssl:
+    certificate_authorities: ['/etc/ca.crt']
+    supported_protocols: ["TLSv1.0", "TLSv1.1", "TLSv1.2"]
+----
```

Comment:
Preview: [`ssl`](https://observability-docs_2729.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-lightweight.html#monitor-http-ssl)

This option currently links back to the [Heartbeat docs](https://www.elastic.co/guide/en/beats/heartbeat/current/configuration-ssl.html) to help the user understand what a valid `ssl` option looks like. This page is pretty long so I didn't think we should copy it over to these docs (like I did with other [data types](https://observability-docs_2729.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-lightweight.html#synthetics-lightweight-data-types)). Are there any concerns with linking back to the Heartbeat docs here?

Reply:
While I generally think we should avoid this, I'm OK with it here.